File: mentor-on-demand-app-client/src/app/models/skill.ts
export class Skill {

    id: number;
    name: string;
    description: string;
    prerequisites: string;

    constructor(id: number, name: string, description: string, prerequisites: string) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.prerequisites = prerequisites;
    }

    // get Id(): number {
    //     return this.id;
    // }

    // set Id(id: number) {
    //     this.id = id;
    // }

    // get name(): string {
    //     return this._name;
    // }

    // set name(name: string) {
    //     this._name = name;
    // }

    // get description(): string {
    //     return this._description;
    // }

    // set description(description: string) {
    //     this._description = description;
    // }

    // get prerequisites(): string {
    //     return this._prerequisites;
    // }

    // set prerequisites(prerequisites: string) {
    //     this._prerequisites = prerequisites;
    // }

}

File: mentor-on-demand-system-admin-ms/src/main/java/com/ibm/mods/admin/model/Skill.java
package com.ibm.mods.admin.model;

public class Skill {
	
	private long id;
	private String name;
	private String description;
	private String prerequisites;
	
	public long getId() {
		return id;
	}
	
	public void setId(long id) {
		this.id = id;
	}
	
	public String getName() {
		return name;
	}
	
	public void setName(String name) {
		this.name = name;
	}
	
	public String getDescription() {
		return description;
	}
	
	public void setDescription(String description) {
		this.description = description;
	}
	
	public String getPrerequisites() {
		return prerequisites;
	}
	
	public void setPrerequisites(String prerequisites) {
		this.prerequisites = prerequisites;
	}

}
